bpo-44926: `get_type_hints`: Add note about type aliases with forward refs (#27859)
python/cpython: Doc/library/typing.rst

           'name': Annotated[str, 'some marker']
       }

   .. note::

      :func:`get_type_hints` does not work with imported :ref:`type aliases <type-aliases>` that include
      forward references.

>       Consider avoiding forward references by enabling postponed evaluation of annotations (:pep:`563`) instead.

Reviewer: Thanks for the docfix. One minor nit: 
```suggestion
      Enabling postponed evaluation of annotations (:pep:`563`) may remove the need for most forward references.
```
because [PEP 563 doesn't obviate](https://www.python.org/dev/peps/pep-0563/#forward-references) all forward references.

Also please limit your lines to 80 characters, following the [style guide](https://devguide.python.org/documenting/#use-of-whitespace). Thank you!
